### DISCUSSION TITLE
Serious errors in the search function

### DISCUSSION THREAD

**Discussion by DreamOfIce** [mdn/mdn-community#281] [Content] [answered]
Today, when I searched for something in mdn, I got the following error:
![CAC724F8-0D8D-4BED-98C3-48249832586A](https://user-images.githubusercontent.com/80821706/201014887-972cec44-8fe7-4d87-bca5-b24000a48f73.jpeg)
My default language is simplified Chinese (zh-cn), the error message has a zn-cn seems somewhat similar, when I change the url in the zh-cn to zn-cn, the search worked. And returned the results in simplified Chinese.
This function was working fine a few days ago, I'm not sure if this is an accident or a planned change? But in any case, this error should not appear after clicking the search button, I hope it will be fixed soon

**Comment by yin1999** [comment, marked as answer]
Hi @DreamOfIce, thanks for raising this. I think this is [a typo error in mdn/rumba](https://github.com/mdn/rumba/blob/4c3992aa978fcc62af68eb418b4462c27b5ad720/src/api/elastic.rs#L68-L69) (which is the backend of search api). I have created a [PR](https://github.com/mdn/rumba/pull/102) to fix it.

**Reply by yin1999** [reply]
The above fix has been deployed. 🙂

![image](https://user-images.githubusercontent.com/15844309/201461290-3f14587d-602f-4bfd-8e1b-609326f6e563.png)
